sdk/src/services/Erc20Service.ts

```diff
@@ -1652,6 +1687,7 @@ export class Erc20Service {
             headers: {
                 'x-backend-wallet-address': xBackendWalletAddress,
                 'x-idempotency-key': xIdempotencyKey,
+                'x-transaction-mode': xTransactionMode,
                 'x-account-address': xAccountAddress,
```

Comment:
Mam so much repetition to add a header. Feels like it should be centralized

Reply:
Fully agreed. My hono + zod refactor fixes a bunch of these issues